contracts/OpenSTValueMock.sol

```diff
@@ -28,7 +28,14 @@ import "./OpenSTValue.sol";
 contract OpenSTValueMock is OpenSTValue {
 	uint256 private constant BLOCKS_TO_WAIT_LONG = 8;
 	uint256 private constant BLOCKS_TO_WAIT_SHORT = 5;
-
+	uint256 private constant intentsMapIndexPosition = 4; 
```

Comment:
This variable should be in OpenSTValue.sol as platform will also need it. They will query from OpenSTValue.sol through interaction layer.
intentsMapIndexPosition => intentsMappingStorageIndexPosition

Reply:
Done!